concepts/05 Widgets/Tabs/15 Control the Behavior.md

```diff
@@ -1,64 +1,420 @@
 An end user can select **Tabs** items in two different modes: *'single'* (by default) or *'multiple'*. You can use the [selectionMode](/api-reference/10%20UI%20Widgets/dxTabs/1%20Configuration/selectionMode.md '/Documentation/ApiReference/UI_Widgets/dxTabs/Configuration/#selectionMode') option to change the mode. 
 
+---
+##### jQuery
+
+    <!--HTML-->
+    <div id="tabsContainer"></div>
```

Comment:
HTML is not needed because it distract the reader from what we are demonstrating. It is generally understood that there's a container for the widget.